src/main/kotlin/no/nav/familie/ef/sak/repository/domain/BehandlingJournalpost.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.Table
+
+@Table("behandling_journalpost")
+data class BehandlingJournalpost(@Column("journalpost_id") val journalpostId: String,
```

Comment:
Hvis vi skal følge naming convention som vi diskuterte om på slack så blir vel dette
`@Table(behandlingsjournalpost)`, `Behandlingsjournalpost` ? 
 https://nav-it.slack.com/archives/G012YEK9YQ1/p1602659136075400